python/lsst/ts/standardscripts/maintel/laser_tracker/align.py

```diff
@@ -0,0 +1,206 @@
+    - "M2 Hexapod aligned with laser tracker.": M2 aligned.
+    - "Camera Hexapod aligned with laser tracker.": M2 aligned.
+    """
+
+    def __init__(self, index: int, add_remotes: bool = True):
+        super().__init__(index, descr="Align MTCS components with laser tracker.")
+
+        self.laser_tracker = RemoteGroup(
+            domain=self.domain,
+            components=["LaserTracker:1"],
+            intended_usage=None if add_remotes else Usages.DryTest,
+            log=self.log,
+        )
+
+        self.mtcs = MTCS(
+            domain=self.domain,
+            intended_usage=None if add_remotes else MTCSUsages.DryTest,
+            log=self.log,
+        )
+
+        self.timeout_align = 120
+        self.timeout_std = 60
+
+        self.max_iter = 10
+        self.tolerance_linear = 1.0e-7 #meter
+        self.tolerance_angular = 5.0 / 3600.0 #degrees
+        self.target = 'M2'
+
+    @classmethod
+    def get_schema(cls):
+        url = "https://github.com/lsst-ts/"
+        path = (
+            "ts_externalscripts/blob/main/python/lsst/ts/standardscripts/"
+            "maintel/laser_tracker/align.py"
+        )
+        schema_yaml = f"""
+        $schema: http://json-schema.org/draft-07/schema#
+        $id: {url}{path}
+        title: MaintelLaserTrackerAlign v1
+        description: Configuration for Maintel laser tracker alignment SAL Script.
+        type: object
+        properties:
+            max_iter:
+                description: maximum number of iterations to align components.
+                type: number
+                default: 10.0
+                minimum: 1.0
+            target:
+                description: >-
+                  Target to align. Options are:
+                    M2: Secondary mirror.
+                    Camera: LSST Camera.
+                type: string
+                enum: ["Camera", "M2"]
+                default: M2
+            tolerance_linear:
+                description: tolerance for rigid body degrees of freedom corrections (meters).
+                type: number
+                default: 1.0e-7
+                minimum: 0.0
+            tolerance_angular:
+                description: tolerance for tip/tilt degrees of freedom corrections (deg).
+                type: number
+                default: 0.00138
+                minimum: 0.0
+        additionalProperties: false
+        """
+        return yaml.safe_load(schema_yaml)
+
+    async def configure(self, config):
+        self.max_iter = config.max_iter
+        self.target = getattr(LaserTrackerComponent, config.target)
+        self.tolerance_linear = config.tolerance_linear
+        self.tolerance_angular = config.tolerance_angular
+
+    def set_metadata(self, metadata):
+        """Set estimated duration of the script."""
+
+        metadata.duration = (self.timeout_align + self.timeout_std) * self.max_iter
+
+    async def align_target(self, target):
```

Comment:
So, `target` is a class member, you can probably use it instead of having it as a method parameter.

Reply:
Done